src/Rollbar.elm

```diff
@@ -16,14 +19,14 @@ module Rollbar exposing (Environment, Level(..), Rollbar, Scope, Token, environm
 
 import Bitwise
 import Dict exposing (Dict)
-import FNV
 import Http
 import Json.Encode as Encode exposing (Value)
+import Murmur3 as FNV
```

Comment:
Switching to the `Murmur3` package seems fine, but let's not alias the import to `FNV` - FNV is a totally different library from Murmur3, which would not occur to me if I were reading the code and had not scanned through the imports! 😅